### DISCUSSION TITLE
Property 'provider' does not exist on type '{ readonly chains: { readonly id: 43114; readonly name: "Avalanche"; readonly network: "avalanche"; readonly nativeCurrency: { readonly decimals: 18; readonly name: "Avalanche"; readonly symbol: "AVAX"; }; readonly rpcUrls: { ...; }; readonly blockExplorers: { ...; }; readonly contracts: { ...; }; }[]; readonly publ...'.

### DISCUSSION THREAD

**Discussion by Harsh22222** [rainbow-me/rainbowkit#1246] [Feedback]
I am getting this error while I am trying to integrate the rainbow wallet in my project.

` provider } = configureChains([avalanche],  [
  alchemyProvider({ apiKey: "APIKEY"}),
  publicProvider()
]);
`

**Comment by DanielSinclair** [comment]
If you're still seeing this issue, please open a new Issue with a reproducible example and we can investigate.
